File: MiniDOMWebCtrls/DMD/Office/GPSRecordsHandler.cs

namespace minidom.Forms
{
    public class GPSRecordsHandler : CBaseModuleHandler
    {
        public GPSRecordsHandler() : base(ModuleSupportFlags.SCreate | ModuleSupportFlags.SDelete | ModuleSupportFlags.SEdit | ModuleSupportFlags.SAnnotations | ModuleSupportFlags.SDuplicate)
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            return new Office.GPSRecordCursor();
        }
    }
}

File: MiniDOMWebCtrls/DMD/Office/GPSRecordsHandler.vb
Imports minidom
Imports minidom.Forms
Imports minidom.Databases
Imports minidom.WebSite

Imports minidom.Office
Imports minidom.Sistema
Imports minidom.Anagrafica

Namespace Forms
 
    Public Class GPSRecordsHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            MyBase.New(ModuleSupportFlags.SCreate Or ModuleSupportFlags.SDelete Or ModuleSupportFlags.SEdit Or ModuleSupportFlags.SAnnotations Or ModuleSupportFlags.SDuplicate)
            
        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Return New GPSRecordCursor
        End Function

    End Class


End Namespace